sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionTestBase.scala

```diff
@@ -295,9 +295,6 @@ abstract class StatisticsCollectionTestBase extends QueryTest with SQLTestUtils
     }
   }
 
-  def getDataSize(file: File): Long =
```

Comment:
Why this change?

Reply:
Move `StatisticsCollectionTestBase.getDataSize` to `SQLTestUtils.getLocalDirSize`.

Reply:
But I can see the same code in StatisticsCollectionSuite.scala

Reply:
Let's not remove this function to avoid confusion, @wangyum .